src/github/client/github-queries.ts

```diff
@@ -9,6 +9,42 @@ query {
 	}
 }`
 
+type RepositoryNode = {
+  node: Repository
+}
+
+export type GetRepositoriesResponse = {
+  viewer: {
+    repositories: {
+      pageInfo,
+      edges: RepositoryNode[]
+    }
+  }
+};
+
+export const GetRepositoriesQuery = `query ($per_page: Int!, $cursor: String) {
```

Comment:
Are we going to switch to GQL at some point?

Reply:
eventually... I guess it comes down to when we think GHs graphql is robust enough